app/views/payments/index.html.erb

```diff
@@ -0,0 +1,29 @@
+<section class='page'>
+    <nav class="navbar">
+        <ul class='s-ul'>
+            <li><p><%= link_to authenticated_root_path do %><p class='back'>&#x2039;</p><% end %></p></li>
+            <li><p><%=@group.name %></p> </li>
+            <li> 
+                <%= button_to "Out", destroy_user_session_path, method: :delete, class: "btn-out" %>
+            </li>
+        </ul>
+    </nav>
+
+    <div class='c-container'>
+        <ul class='c-ul'>
+            <% @payments.each do |payment| %>
+                    <li class='c-li'>
+                        <div class='c-img' >
+                        </div>
+                        <div class='c-info'>
+                            <div class='c-name'><p><%= payment.name %></p></div>
+                            <div class='c-amount'><p>$ <%= payment.amount %></p></div>
+                        </div>
+                    </li>
+            <% end %>    
+        </ul>
+    </div>
```

Comment:
- [ ] [optional] @vikas924 Great work implementing the changes suggested, I would also recommend you consider adding total amount of the transactions at the top of the page before the transactions.